dist/ld/latest/eagle.flash.1m0m1s.ld

```diff
@@ -11,9 +12,9 @@ MEMORY
   irom0_0_seg :                         org = 0x40201010, len = 0xf9ff0
 }
 
-PROVIDE ( _SPIFFS_start = 0x402FB000 );
-PROVIDE ( _SPIFFS_end = 0x402FB000 );
+PROVIDE ( _SPIFFS_start = 0x402fb000 );
+PROVIDE ( _SPIFFS_end = 0x402fb000 );
```

Comment:
Before you merge this, can you add the _FS_* aliases here as well so it also builds with core git?

Reply:
```diff
-INCLUDE "../ld/eagle.app.v6.common.ld"
+PROVIDE ( _FS_start = _SPIFFS_start );
+PROVIDE ( _FS_end = _SPIFFS_end );
+PROVIDE ( _FS_page = _SPIFFS_page );
+PROVIDE ( _FS_block = _SPIFFS_block );
+
+INCLUDE "../ld/local.eagle.app.v6.common.ld"
```
?

Maybe Core needs a ESP::getFlashLayout() or something? I tried looking at other projects, ESPEasy for example defines global flag at runtime to switch between _SPIFFS_... and _FS.... Tasmota hides them under ARDUINO_RELEASE... flags. 2.6.0 is due in september, maybe we can figure something out before that.

Reply:
Also, see https://github.com/xoseperez/eeprom_rotate/issues/6 (cc @reaper7)
I will fix pio builds, but not IDE. I don't think IDE builder will search in the project for ldscripts, so it kind of annoying thing to manage already, both INCLUDE and this. And until IDF (partitions.csv) switch it probably will be :/